Cargo.toml

```diff
@@ -6,7 +6,9 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-rocket = "0.5"
+rocket = { version = "0.5.1", features = ["json"] }
 rocket_cors = "0.6"
 tempfile = "3"
 log = "0.4"
+env_logger = "0.9"
+serde = { version = "1.0", features = ["derive"] }
```

Comment:
Missing newline at end-of-file. please make sure to maintain it in all edited files.